README.markdown

```diff
@@ -51,6 +51,15 @@ In other words, let the machine do most of the work for you.
 * Quick edit of scans
 * PDF support
 
+## Test with Docker
```

Comment:
Please do not put these instructions in the README. Instead, please create a sub-page like doc/install.docker.markdown for instance, and just put a link to it in the Installation section.